### PR TITLE
Hopefully fix site workflow

### PR DESCRIPTION
**Pull request by kubukoz**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -94,5 +94,7 @@ ThisBuild / githubWorkflowAddedJobs += WorkflowJob(
         "publish_dir" -> "./target/website",
         "github_token" -> "${{ secrets.GITHUB_TOKEN }}"
       )
-    )
+    ),
+  scalas = List(Scala213),
+  javas = githubWorkflowJavaVersions.value.toList
```
Comment (kubukoz) [inline build.sbt:99]: man couldn't decide if he prefers Seq or List so there's both in the API :D @djspiewak

Comment (djspiewak) [reply build.sbt:99]: Seq for sbt (where you get the appenders), and List for everything else!

Comment (kubukoz) [reply build.sbt:99]: you don't get appenders for List? 🤔

Comment (kubukoz) [reply build.sbt:99]:
```scala
val x = settingKey[List[String]]("foo")
x := List("hello")
x += "world"
```

seems to work:

```
sbt:root> show x
[info] * hello
[info] * world
```

Comment (djspiewak) [reply build.sbt:99]: Hmm. That *didn't* work when I was building the plugin. I wonder if it was later fixed?

Comment (kubukoz) [reply build.sbt:99]: Possible :) although https://github.com/sbt/sbt/blame/develop/main-settings/src/main/scala/sbt/Append.scala hasn't been modified in years

**Pull request merged**